buildlib/azure-pipelines-pr.yml

```diff
@@ -265,6 +265,65 @@ stages:
             inputs:
               pathToPublish: '$(Build.ArtifactStagingDirectory)'
               artifactName: ucx_bin_$(Build.BuildId)
+      - job: Test_in_containers
+        dependsOn: Distros
+        displayName: UCX tests in containers
+        pool:
+          name: MLNX
+          demands:
+          - ucx_docker -equals yes
+        strategy:
+          matrix:
+            ubuntu20:
+              CONTAINER: ubuntu2004
+        steps:
+          - checkout: none
+          - task: DownloadBuildArtifacts@0
+            displayName: 'Download Build Artifacts'
+            inputs:
+              artifactName: ucx_bin_$(Build.BuildId)
+              downloadPath: $(System.DefaultWorkingDirectory)
+          - bash: chmod u+rwx $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId) -R
+          - bash: |
+              set -Eex
+              ./bin/ucx_perftest -t tag_bw &
+              server_pid=$!
+              sleep 1;
+              ./bin/ucx_perftest -t tag_bw localhost 2>&1 | tee perf.txt
+              wait $server_pid
+              grep "Final:" perf.txt
+            displayName: Test ucx_perftest
+            workingDirectory: $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install
+            env:
+              LD_LIBRARY_PATH: $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/lib:$LD_LIBRARY_PATH
+          - bash: |
+              set -Eex
```

Comment:
1. maybe put this in a helper script? it it's pretty big
2. better not mix ucx_info and client_server: first run both ucx_info, then client_server
3. can we separate the stages into more clear parts (by comments/prints for example)
- ucx_info -v
- client v1.11 with new server
- new client with server v1.11